path-finding.asciidoc

```diff
@@ -12,3 +12,31 @@ Relevant questions to answer:
   * What information can be sent to intermediate and the final node aside from the critical routing data?
   * What are multi-hop locks? What addition privacy and security guarantees to they offer?
   * How can the flexible onion space be used to enabled packet switching in the network?
+
+
+=== What is "Source-Based" routing and why does the Lightning Network use it?
+
+Source-based routing is a method of path-finding where the sender (i.e. the source) plans the route from itself, through the intermediary nodes, and finally to the destination.
+Once a route has been selected, the sender sends the payment to the first intermediary node, who sends it to the second intermediary node and so on, until it reaches the destination.
+While a payment is in-flight along a route, the route cannot be changed by any of the intermediary nodes, even if a shorter path or a cheaper path (in terms of routing fees) exists.
+
+The Lightning Network uses source-based routing at the time of writing in order to protect user privacy.
+As discussed in the chapter on Onion Routing, the intermediary nodes transmitting the payment are not aware of the full route of the payment; they only know the node they received it from and the node they are sending it to.
+As such, intermediary nodes cannot assist with path-finding because this requires them to know the destination of the payment.
```

Comment:
again not sure if this is true because the current system allows techniques like JIT routing

Reply:
Hmmm.... not sure how to word it in this case. Happy for you to suggest an alternative or should I just kill these few lines?

Reply:
I will merge and afterwards just delete those lines here. I think for now this is the more elegant way